net/lp_rpc.proto

```diff
@@ -12,6 +12,13 @@ service Orchestrator {
   rpc Ping(PingPong) returns (PingPong);
 }
 
+service AIWorker {
+
+  // Called by the aiworker to register to an orchestrator. The orchestrator
+  // notifies registered aiworkers of jobs as they come in.
+  rpc RegisterAIWorker(RegisterAIWorkerRequest) returns (stream NotifyAIJob);
```

Comment:
RegisterAIWorker request doesn't need to be a newly defined type

Reply:
A new `RegisterAIWorkerRequest` was added because the generic `capacity` field is not helpful when trying to manage GPUs for AI jobs in my opinion.  For transcoding one GPU can do multiple requests at a time and there was one job type. For AI, my experience is most models slow down significantly when more than one request is fed to it concurrently. 

Do you think a generic capacity field set at launch of the ai-worker would let the orchestrator appropriately manage the ai workers?

Do you think that AI workers and remote transcoders would always have the same requirements when connecting to the orchestrator?